modules/settings/settings.php

```diff
@@ -158,28 +118,78 @@ public function helper_option_custom_post_type( $args = array() ) {
 		foreach ( $all_post_types as $post_type => $title ) {
 			echo '<label for="' . esc_attr( $post_type ) . '">';
 			echo '<input id="' . esc_attr( $post_type ) . '" name="'
-			. esc_attr( $this->module->options_group_name ) . '[post_types][' . esc_attr( $post_type ) . ']"';
-			if ( isset( $this->module->options->post_types[ $post_type ] ) ) {
-				checked( $this->module->options->post_types[ $post_type ], 'on' );
+			. esc_attr( OptionsUtilities::get_module_options_key( self::SETTINGS_SLUG ) ) . '[post_types][' . esc_attr( $post_type ) . ']"';
+			$the_post_types = OptionsUtilities::get_module_option_by_key( self::SETTINGS_SLUG, 'post_types' );
+			if ( isset( $the_post_types[ $post_type ] ) ) {
+				checked( $the_post_types[ $post_type ], 'on' );
 			}
-			// Defining post_type_supports in the functions.php file or similar should disable the checkbox
-			disabled( post_type_supports( $post_type, $this->module->post_type_support ), true );
```

Comment:
Noting that this needs to be added back in. Right now it was half baked in trunk, so it didn't make sense leaving it in imo.

Reply:
Can we make a separate issue for this? I'm not clear on what's baked/half-baked/unimplemented as well.

Reply:
Yip, I've made #60